### ISSUE TITLE
Collaboration Cycle Checklist for Authenticated Experience (My VA) for #31611 My VA Enhancements - Payments

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#33299]
## Instructions

<details>

Copy/Paste the below into a new issue to create a checklist of required touchpoints for your project after receiving feedback from your Collab Cycle Kickoff.   The checklist and associated teams are current as of **Nov 3, 2021** based on [VSP's Collab Cycle Touchpoints](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Touchpoints.1768423528.html)
- [Platform website Collab Cycle FAQs](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/FAQs.1864073263.html)
- [Collab Cycle Change History](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Change-history.1846870032.html)
  
### How to Configure This Checklist
Tag with vsa and team name

</details>

---

# Collab Cycle Checklist for Authenticated Experience (My VA) for My VA Enhancements - Payments #31611

## List your team, project contributors, and reviewers
<details>

_Record key players so you can quickly copy/paste into requests and your product documentation as your project progresses_

### Product Team
- OCTO-DE Product Lead: Samara-Strauss
- Product Manager: @hdjustice
- FE Engineer: @tmitchellgcio 
- BE Engineer: @lihanli 
- Designer: @tressaellen, Liz Lantz, Angela Agosto
- Researcher: @tressaellen, Liz Lantz, Angela Agosto
- Slack channel: #vsa-authd-exp
- GitHub tag: my-va-dashboard

### VFS participants (when applicable):
- Design Lead: Ryan Thurwell
- Design
- Accessibility: @joshkimux, @angelafowler82
- Analytics: Jenna Ouellet
- IA: @mnorthuis
- Content: @mnorthuis
- QA: @tlei123
- Privacy & Security:  Troy Mosher

</details>

## List project artifacts for requests

<details>

- [Project epic # My VA 2.0 Enhancements #31611](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31611)
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/README.md)
- [Whiteboard sketch or other lo-fi prototypes or wireframes - Combined with On-site Notifications](https://app.mural.co/t/vsa8243/m/vsa8243/1638500075560/0e849f61e282ebe2d8e7119eea504e4c05b0bbe4?sender=9447891e-c22c-4863-8217-3a1273c6c95f)
- ~User flows~
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/discovery-and-research/user-research/payment-information-research-plan.md)
- [Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/discovery-and-research/user-research/payment-information-conversation-guide.md)
- [North Star and KPIs for the product](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/payment-history#measuring-success)
- [Product URL](https://www.va.gov/my-va)
- [GH Test Cases document](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-myva-payment-info.md)
- Finalized design mockup - [Mural](https://app.mural.co/t/vsa8243/m/vsa8243/1640028268317/2ba4e1b2846c87f5d0bfc6bcb643bef84720bea1?sender=408509fb-10d1-4b2f-a6f4-2e2ba827f257) and [Sketch](https://www.sketch.com/s/418ca6f6-cc24-4496-82aa-9deda03b08f6)
- [UXPIN Prototype](https://preview.uxpin.com/45251888896c8dd47ef62aa20d3a89a334726ac1#/pages/145483168/simulate/no-panels?mode=i)
- ~Content brief~
- [Regression test plans](https://dsvavsp.testrail.io/index.php?/suites/view/5&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=3375)
- Product test [**case**](https://dsvavsp.testrail.io/index.php?/cases/view/14393)/test [**plan**](https://dsvavsp.testrail.io/index.php?/plans/view/2824) in TestRail with test execution logs
- [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/297) 
- [Summary(Defects) reports in TestRail](https://dsvavsp.testrail.io/index.php?/reports/view/296) - NO DEFECTS
- [Accessibility staging review prep document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/Staging%20Review/Accessibility%20testing.md)
- Technical diagrams (architecture diagram, sequence diagram)
     - [Frontend documentation ](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/payment-history/documentation)
     - [API flows diagram](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/documentation/Payments%20and%20Debts%20diagram.jpg)
- ~Any new publicly-exposed endpoints~
- ~Any new interactions with dependent VA backends~
- [Release plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/product/release-plan.md)
- [Product guide for contact center](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/product/product-guides.md)

</details>

## Pre-Launch touchpoints linked to VSP reference info + practice areas involved at each step

<details>

- [x] Collaboration Cycle Kickoff | [Request #33468 ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33468)

- [x]  Design Intent  |  [Request #33477](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33477)  |  [Feedback 34047](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34047) | Mtg occurred 12/13 3:30pm ET
  - Content [Sitewide Content and IA intake request #34015](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34015)  - Feedback from Mikki Northuis would like to continue talking about:
      - Labeling for the section as multiple teams continue to figure out this idea of "financial things being together"
      - Placement of travel reimbursement and copays
      - How/if this payment section evolves when Education section is created
  - Design
  - OCTO
  - IA | [Feedback from Jim](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33477): Thanks @hdjustice and @tressaellen. It looks like you have the IA concerns well-integrated into your design and research plans, so I have no other feedback. Great work! | 

- [x] Research Plan Review |  [ticket #64](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/64)
  - OCTO

- [x] Analytics Request | [Link Clicks Request #37589](https://github.com/department-of-veterans-affairs/va.gov-team/issues/37589) and [API load success/fail Request #38739](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38739#issuecomment-1072542571)
  - OCTO

- [x] Midpoint Review |  [#34549 Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34549) | [QA Feedback Ticket #35100](https://github.com/department-of-veterans-affairs/va.gov-team/issues/35100) | additional feedback below (Midpoint section below)
  - Accessibility |  [#issuenum]
  - Content
  - Design - [No feedback comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34549#issuecomment-1010158186)
  - OCTO
  - IA
  - QA - [Feedback ticket #35100](https://github.com/department-of-veterans-affairs/va.gov-team/issues/35100)

- [x] Staging Review | [Request #38376](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38376)
  - [Accessibility Testing Prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/Staging%20Review/Accessibility%20testing.md) 
  - Content
  - Design
  - OCTO
  - IA
  - QA | [Testing Ticket #35149](https://github.com/department-of-veterans-affairs/va.gov-team/issues/35149) - 100% passed with NO DEFECTS!

- [x] Privacy and Security Review | [Request #464](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/issues/434) - Approved with No feedback!

- [x] Contact Center Review |  [Request #38181](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38181)
    - Contact Center - **[No Feedback from Contact Center/VEO reviews](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38181#issuecomment-1068505372)**
 
</details>

## Post-Launch Tasks

<details>

- [ ]  Update [Launch Tracker on GitHub](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/vsa_launch_tracker.md)

- [ ]  Post @here announcement in your Slack channel after launch

- [ ] ~Full accessibility and 508 office audit | [Overview](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Full-accessibility-and-508-office-audit.1774846068.html) | [Request template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=noahgelman%2C+shiragoodman&labels=508%2FAccessibility%2C+vsp-product-support%2C+collaboration-cycle%2C+collab-cycle-review&template=full-accessibility-and-508-office-audit.md&title=Full+Accessibility+%26+508+Office+Audit+%5BTeam+Name+-+Feature+Name%5D) | [#issuenum]~
   - ~Accessibility~

- [ ] Post-Launch Check-in | [Overview](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Post-launch-check-in.1781956691.html) |  [#issuenum]
  - Product

</details>

## Other Resources

<details>

- [Product Development Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/Product%20Development%20Checklist.md)

# Product development checklist

### Ask yourself the questions below weekly, starting from Day 1.

This checklist is for you to gut check that you've taken all of the steps necessary for your feature to be successful for users. You won't have answers to all of these questions right away, and some you'll want to take action on multiple times, but you should check in to make sure these things are happening iteratively over the lifetime of your product.

This resource is meant to be used in parallel with the [VSP Collaboration Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/working-with-vsp/vsp-collaboration-cycle). The items below that correspond with one or more VSP collaboration points are starred `*`. The VSP collaboration cycle is designed to guide your team through successful development but it is still your responsibility to ensure that all the points below are sufficiently met.

_Note: as always, use the [#vfs-platform-support](https://dsva.slack.com/channels/vfs-platform-support) Slack channel whenever you have a question on how to proceed!_

---

### ...usable (for ALL users)?

- [ ] Usability testing indicates new changes improve / don't worsen back-of-house VA employee experience
- [ ] [Usability testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/lead-researcher-checklist.md) indicates new changes enable users to do what they need to do `*`
- [ ] Everything is [508 Accessible](https://github.com/department-of-veterans-affairs/va.gov-team/blob/de1d32ebfc17d4b887133e33d946ece5a12913d8/platform/accessibility/README.md) and there are no Sev 1 accessibility bugs `*`
- [ ] Your content adheres to the [VA.gov content style guide](https://design.va.gov/content-style-guide/link) and meets [plain language best practice guidance](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/content/content-best-practices.md) `*`
- [ ] Everything is consistent with VA's [Design System](https://design.va.gov/) `*`
- [ ] Downtime notifications and error messages:
  - [ ] [Error message verbiage](https://design.va.gov/patterns/messaging-dictionary)
  - [ ] [Error message style guide](https://design.va.gov/patterns/messaging-error-messages)
  - [ ] [Error message configuration instructions](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/downtime-notifications/)
- [ ] [End-to-end manual QA](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/README.md) is complete and no Sev1 bugs `*`
- [ ] [End-to-end UAT](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/what-is-uat.md)(with real users) complete and no Sev1 bugs
- [ ] Availability and latency targets are defined `*`

### ...findable?

- [ ] [URL, nav, bread crumbs, and cross-links](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-best-practices.md) are consistent with VA's style guide and map to the user's mental model `*`
- [ ] [SEO](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/seo-best-practices.md) is baked into everything
- [ ] Static entrance/intro content is in place `*`
- [ ] [Redirects](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md) (if necessary) from legacy tools/pages are in place

### ...compliant w/VA official stuff?

- [ ] Changes adhere to DEPO's [7 Digital Modernization Principles](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-gov-relaunch-2018/va-gov-strategy/digitalmodernizationstrategy.pdf)
- [ ] Changes have appropriate ATO

### ...measurable?

- [ ] [KPIs are defined and tracked](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/insights-analytics/kpi-tracking/kpi-process.md)
- [ ] Feature is included in the weekly Product Health Report
- [ ] [Google Analytics / Google Tag Manager](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/google-analytics/analytics-request-guide.md#requesting-google-analytics-implementation-and-qa) setup is complete `*`
- [ ] Phased, data-informed [release plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/97759a81a47c73da8bf03e35f3a13bb3c689d18b/platform/product-management/release-plan-template.md) is complete `*`
- [ ] Monitoring is setup correctly and working `*`

### ...secure and reliable?
- [ ] Users' private data is safe, including not logging PII `*`
- [ ] Feature doesn't introduce new vulnerabilities `*`
- [ ] Playbook exists on most likely failure modes and how to respond quickly to issues `*`
- [ ] The feature has 90% [unit test coverage](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/new-unit-test/), running in CI/CD
- [ ] Feature can handle anticipated load and traffic spikes, with [a completed load test](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/load-testing/README.md)
- [ ] Rate limits are set

### ... supportable?
- [ ] Dashboards and alerts are set up for finding and investigating issues
- [ ] Point of contact are documented for the app itself and any new backend dependencies `*`
- [x] There's a [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/34add7c7b3d558158ccf3f599e79c2380076481c/platform/product-management/product-outline-template.md)
- [ ] [Contact centers know how to help users who call in about the feature](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md) `*`

### ...marketable?
- [ ] You've done a [Go/No Go meeting](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/go-no-go-meeting-template.md)

</details>

# Collab Cycle Touchpoints - Feedback
## Collab Cycle Kickoff meeting

<details>
Hi @hdjustice. Thanks for completing this Collaboration Cycle Kickoff ticket! Based on your responses, your project is an iteration to a tool and/or application so we are recommending the following Collaboration Cycle touchpoints:

Design Intent
Analytics Request
Midpoint Review
Contact Center Review
Staging Review
Privacy and Security Review

**Post Launch**:
Full Accessibility and 508 Office Audit
Post-launch Check-in

**The following practice areas** will be included throughout the Collaboration Cycle process:
Design
Content
Information Architecture (IA)
Quality Assurance (QA)
Accessibility
Product

We understand that scope of work and/or solutions can change as you work through your design process. If you find that your scope and/or solution changes, please reach out so that we can update and/or modify your touchpoints appropriately.

Your project has been added to the Platform Collaboration Point Tracker. When you’re ready to schedule your recommended Collaboration Cycle touchpoints, please review the guidance outlined on the Collaboration Cycle Touchpoints page.
</details>

## Design Intent meeting

<details>

- Active Links with colored backgrounds, not yet approved styles. Needs to be[ introd as an experimental design](https://design.va.gov/documentation/contributing-to-the-design-system)
- Icon being used is an error alert, should use a different one?  We are fine to use whatever icon is suggested by VSP
- Links (payments and debt) need to be more descriptive for users to understand where they will go
- (TF) Payment Information name is flex, we want to be consistent with other teams' products
- Mobile team card sorts, how will this influence our work? mobile team ran card sort to see how people organize their info
- Planning to do pretesting? Yes, we are planning on usability testing to researching the section, placement and info to include.
- (TF) Further discussions will be held to keep in mind the future of My VA and what that will look like.
- Josh would love a sync up

Ticket #34047 assigned with the below feedback:

> **MUST**: The active links with colored (yellow, blue, and gray) backgrounds are not an approved link style. Remove the background color. Anything new will need to go through the experimental design process.
>
> **SHOULD**: Consider changing error "!" icon in front of messages and payment sections to warning "!" icon. Another option could be to remove the icon all together since warning icons typically insinuate something has gone wrong. The warning icon seems applicable for payments that need made however will messages always be in negative consequence of something? See design system alert box section for documentation around this. https://design.va.gov/components/alertboxes#warning-alert

</details>

## Research/Conversation guide Review
<details>

[**Ticket #64**](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/64) submitted and reviewed by Kevin M Hoffman ([Slack thread](https://dsva.slack.com/archives/C909ZG2BB/p1641398079273800?thread_ts=1641389581.253900&cid=C909ZG2BB)):
>I've reviewed both the convo. guide and the plan; overall looks great! My only point of feedback is that your main hypothesis isn't worded in a way that it clearly describes the behavior you're looking to observe, but I can see you'll get at those behaviors clearly from the conversation guide "what to watch for" sections. You can modify the hypothesis if you like, I'll submit this study to Perigean to begin recruiting later today.

</details>

## Experimental Design meeting
<details>

Link to [meeting notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/Meeting%20Notes/Mtg%20Notes%201-7-22%20-%20Design%20System%20Council%20(Experimental%20Design%20proposal%20review).md)

**Decision** 
- Want to hear what you learn
     - Especially related to
          - Perceivability
          - Comprehension
          - Trauma triggers

</details>

## Midpoint Review
<details>

**_1/11/2022_**

**Design - Allison** 
- We may be missing a period in a line break in Claims and Appeals section

**Design/AI - Ryan/Martha/Jim** 
- Concerns were already brought up during design council meeting (mostly on On-site Notifications)
- Possible triggering (trauma) of the warning pattern
- Can they be more descriptive for context (if in-line notification is taken out of the section, does it make sense)? 
- Can they be grouped together (in-line notifications and on-site notifications)?
- Colorblindness, pattern of icon-text-icon-text can seem to have equal weight, nothing really jumps out

**Accessibility - Noah**
- Screenreader needs to announce correctly (Angela will be checking this out today)

**QA - Dariusz** 
- Provided Cypress testing link for automated testing
- Test rail cases need to be updated - [Ticket #35100](https://github.com/department-of-veterans-affairs/va.gov-team/issues/35100) assigned

**Comments**
- Research/Convo plan is great! - Jim
- Pilot went really well this morning - Liz

**Any changes on product, comment on Mid-point ticket #34549**
</details>

## Analytics Implementation Review
<details>
Analytics requested are accessible, no issues found.  
Samara requested additional analytics around performance.  New ticket submitted.
</details>

## Contact Center guide Review
<details>
No changes needed from Contact Center/VEO reviews!
</details>

## Staging Review
<details>

My Notes from the review:
> - A11y - alert icons on the page - alt text for those icons 
> - A11y - headings being read out on mobile screen readers comes out weird
> - IA - formatting on claims and appeals status (could be from the data we are received)
> - Content - FE doc Inconsistencies with the layout and presented hierarchy of payments and debts
> - Content - listed out payments, there was types of payments, would be nice on debts
> - No QA feedback!
> - No Design feedback!
> 

Staging Review tickets created - None are MUST DO.  
> - [#39470:  [Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.05.1)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39470)
>      - Josh: I'd recommend collaborating with the designer to explore the initial reasons behind this heading structure. As the total debt line is a one liner, and not something starting a full section, this may be more of a content organization issue (which might be better addressed in future versions of the my va dashboard).
> - [#39471: [Other] No VA.gov Experience Standard for the issue found. (0.00.1) ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39471)
>      - When using IOS with VoiceOver, the heading which informs the user of their most recent deposit is distributed over several lines, as an example, one line contains only the word "on." This creates a tedious experience for the user, who must swipe several times rather than simply putting focus on the heading and hearing the information.
>      - Josh: This is likely due to [text-splitting issues known to occur on VoiceOver[(https://axesslab.com/text-splitting/). We may need to use a workaround of nesting the headings' contents within a <span role="text">.
> - [#39473: [Other] No VA.gov Experience Standard for the issue found. (0.00.2) ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39473)
>      - Using user+39, the user hears "We can't load your claims and appeals right now." If the user swipes left from that, they land on a space where VoiceOver says nothing. I believe that's the "claims and appeals" heading.
>      - Josh: This could be VoiceOver landing on the alert icon. We should test to see if adding alt text to it will address the issue
> - [#39474: [Assistive tech and device support] Pages require sensory characteristics to be understood or operable. (11.04.1) ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39474)
>      - Following VA design system guidance, we recommend including a sr-only span to provide an alternative to this icon font. For example: Alert: [or whatever equivalent alternative needed e.g. "Important"].
> - [#39475: [Content styleguide] The content isn't written consistently across static pages, tools, and devices. (05.11.1)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39475)
>      - Payments/Debts are grouped together, and placed one under the other on the page, yet when Payments amounts are listed, the font appears larger and the heading is not consistent to font/heading/style when Debts exist.
> - [#39476: [Content organization] Your content or feature isn't placed within an appropriate or relevant area of VA.gov. (02.05.2) ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39476)
>      - Place the heading Payments and Debts above the Payment history error warning, or use some other means to give this warning message context.
> - [#39477: [Content styleguide] The content isn't written consistently across static pages, tools, and devices. (05.11.1)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39477) 
>      - Include Type: under Debts for consistency to Benefit Payments and Debt, since these 2 are categorized together and live on the same hierarchal plane.
> - [#39480: [Content organization] Long or complex sections of content aren't broken up into content chunks or bulleted lists. (02.06.1)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39480)
>      - If possible, break individual pieces of actionable information in the "compensation claim received" section into new lines with punctuation.

</details>

## Privacy & Security Review
<details>
Per feedback received from Troy Mosher:

> Platform Security has reviewed this request, has no security concerns or feedback, and recommends approval to proceed.

</details>
